### PR TITLE
Preserve preview information we've already parsed

### PR DESCRIPTION
**Pull request by schwers**
👓  @nramadas @phil303

### CODE REVIEW

File: src/models2/PostModel.es6.js [hunk 1]
```diff
@@ -108,6 +108,8 @@ export default class PostModel extends RedditModel {
 
   static DERIVED_PROPERTIES = {
     expandable(data) {
+      if (data.expandable) { return data.expandable; }
```
Comment (phil303) [inline src/models2/PostModel.es6.js]: do you mind multilining these?

**Comment by phil303** [comment]
Can you expand on the description. I don't completely understand how this fixes the bug

**Comment by phil303** [comment]
:nail_care:

**Comment by schwers** [comment]
:haircut_man:

**Comment by nramadas** [comment]
:fish:

**Comment by uzi** [comment]
:fish:

**Pull request merged**